packages/zostso/src/doc/IASAppResponse.ts

```diff
@@ -0,0 +1,51 @@
+/*
+* This program and the accompanying materials are made available under the terms of the
+* Eclipse Public License v2.0 which accompanies this distribution, and is available at
+* https://www.eclipse.org/legal/epl-v20.html
+*
+* SPDX-License-Identifier: EPL-2.0
+*
+* Copyright Contributors to the Zowe Project.
+*
+*/
+
+import { ITsoMessage } from "../../src";
+
+export interface IASAppResponse {
+    /**
+     * Version in response
+     * @type {boolean}
+     * @memberof IASAppResponse
+     */
+    version: string
+    /**
+     * Data from response
+     * @type {boolean}
+     * @memberof IASAppResponse
+     */
+    tsoData: ITsoMessage[]
+    /**
+     * Reused boolean
+     * @type {boolean}
+     * @memberof IASAppResponse
+     */
+    reused: boolean
+    /**
+     * Data from response
```

Comment:
Starting here, a couple of comments look like copy/paste errors.

Reply:
resolved